src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
same here

Reply:
add transition property )

Reply:
Always add a transition for styles that you want smoothly change